app/src/androidTest/java/ch/epfl/sdp/lobby/global/MockGlobalLobbyRepositoryTest.kt

```diff
@@ -0,0 +1,20 @@
+package ch.epfl.sdp.lobby.global
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import ch.epfl.sdp.db.MockDB
+import org.junit.Assert.*
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MockGlobalLobbyRepositoryTest {
+
+    @Test
+    fun callbackIsCalled() {
+        val glr = MockGlobalLobbyRepository(MockDB())
```

Comment:
It would be better to create the mock repo here to see the elements inside. I cannot confirm this test pass because it the behavior is correct or if the test is erroneous

Reply:
I don't understand your comment